DOC-2181: add fix documentation for TINY-10385 to the 6.8.1 release notes (#3019)
tinymce/tinymce-docs: modules/ROOT/pages/6.8.1-release-notes.adoc

In {productname} 6.8.1 translations were added for this text, though this announcement was still only available in English as the syntax used for the table grid announcement text in detecting strings which are to be translated was invalid.

{productname} 6.8.1 addresses this, now, the syntax has been corrected and each cell in the table grid announces the current selected columns and rows in the correct language, _not only_ in English.

=== The functions schema.isWrapper and schema.isInline didn't exclude element names that started with # those should not be considered elements.
// #TINY-10385
Previously the `isInline` and `isWrapper` functions in the Schema module would incorrectly output `true` for non-element nodes such as `#text` and `#comment`.

As a consequence, in certain cases the editor would attempt to append a child node inside such a non-element node, which is forbidden and results in an error being thrown.

In {productname} 6.8.1, `Schema.isInline` and `Schema.isWrapper` exclude non-element nodes whose names starts with `#`, such as `#text` and `#comment`.

> As a result, these elements are no longer considered as valid inline or wrapper elements, preventing errors caused by element methods being called on non-element nodes.

Reviewer: I'm not sure that `non-element` is correct, nodes that has name that start with `#` are not eligible as `wrapper` element but I'm not sure that the element definition doesn't fit them

> Previously the `isInline` and `isWrapper` functions in the Schema module would incorrectly output `true` for non-element nodes such as `#text` and `#comment`.
> 
> As a consequence, in certain cases the editor would attempt to append a child node inside such a non-element node, which is forbidden and results in an error being thrown.
> 
> In {productname} 6.8.1, `Schema.isInline` and `Schema.isWrapper` exclude non-element nodes whose names starts with `#`, such as `#text` and `#comment`.
> 
> As a result, these elements are no longer considered as valid inline or wrapper elements, preventing errors caused by element methods being called on non-element nodes.

Author: ```suggestion
Previously the `isInline` and `isWrapper` functions in the Schema module would incorrectly output `true` for elements nodes when their name begins with  `#` such as `#text` and `#comment`.

As a consequence, in certain cases the editor would attempt to append a child node inside such a element node, which is forbidden and resulted in an error being thrown.

In {productname} 6.8.1, `Schema.isInline` and `Schema.isWrapper` now, excludes element nodes for those which names starts with `#`, such as `#text` and `#comment`.

As a result, these elements are no longer considered as valid inline or wrapper elements, preventing errors caused by element methods being called on specific element nodes like these.
```

> In {productname} 6.8.1.0 translations were added for this text, though this announcement was still only available in English as the syntax used for the table grid announcement text in detecting strings which are to be translated was invalid.

Author: ```suggestion
In {productname} 6.8.1 translations were added for this text, though this announcement was still only available in English as the syntax used for the table grid announcement text in detecting strings which are to be translated was invalid.
```

> {productname} 6.8.1.1 addresses this, now, the syntax has been corrected and each cell in the table grid announces the current selected columns and rows in the correct language, _not only_ in English.

Author: ```suggestion
{productname} 6.8.1 addresses this, now, the syntax has been corrected and each cell in the table grid announces the current selected columns and rows in the correct language, _not only_ in English.
```
